app/views/overrides/activation_keys/_host_media_type_select.html.erb

```diff
@@ -16,19 +16,29 @@
 %>
 
 <div class="clear-fix" id="media_selection_section">
-  <div class="form-group ">
-     <label class="col-md-2 control-label" for="use_install_media"><%= _('Media Selection') %></label>
+  <div class="form-group">
+     <label class="col-md-2 control-label" for="use_install_media">
+        <%= _('Media Selection') %>
+     </label>
      <div class="col-md-4">
         <label class="radio-inline">
-          <input data-media-selector="true" id="host_use_synced_content" name="media_selector" value="synced_content" type="radio" <%= synced_content_radio  %> <%= synced_content_disabled %>/> <%= _('Synced Content') %>
+          <input data-media-selector="true" id="host_use_synced_content" name="media_selector" value="synced_content" type="radio" 
+            <%= synced_content_radio  %> 
+            <%= synced_content_disabled %>
+          /> 
+          <%= _('Synced Content') %>
         </label>
+        <%= popover("", "To enable the synced content option, this host must use a content source, content view, and lifecycle environment which contain synced kickstart repositories for the selected architecture and operating system.") %>
```

Comment:
This probably should have been translated. oh well :shrug:

Reply:
We can still fix this, @qcjames53 mind popping open a PR with the following change? :

```suggestion
        <%= popover("", _("To enable the synced content option, this host must use a content source, content view, and lifecycle environment which contain synced kickstart repositories for the selected architecture and operating system.")) %>
```
The PR can be a refs commit and get added to the same redmine.